godtools/Views/TractElements/TractElementOperationFunctions.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  TractElementOperationFunctions.swift
+//  godtools
+//
+//  Created by Devserker on 5/1/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import Foundation
+import SWXMLHash
+
+extension BaseTractElement {
+    
+    func buildElementForDictionary(_ data: XMLIndexer, startOnY yPosition: CGFloat) -> BaseTractElement {
+        let dataContent = splitData(data: data)
+        var element:BaseTractElement?
+        
+        if dataContent.kind == "hero" {
```

Comment:
In swift we can switch a string now, which may be a little more readable? What do you think?